test/IRGen/open_boxed_existential_resilience.sil

```diff
@@ -0,0 +1,21 @@
+// RUN: %target-swift-frontend -assume-parsing-unqualified-ownership-sil %s -enable-resilience -emit-ir | %FileCheck %s --check-prefix=CHECK --check-prefix=CHECK-%target-ptrsize --check-prefix=CHECK-%target-runtime
```

Comment:
-enable-resilience won't have the effect of making calls to the stdlib's Error type resilient here, I think. You probably want '// REQUIRES: resilient-stdlib' instead.

Reply:
Thanks! changed that..